docs/index.rst

```diff
@@ -466,6 +466,18 @@ Outputs:
 .. literalinclude:: examples/schema3.json
 
 
+Pydantic models can be set custom root types to support a model that does not have any field exclude ``__root__``.
```

Comment:
Can you make this a separate section of the docs. This is not just about schema generation.

Reply:
I see. I have moved the content to around types sections.